plugins/cvebin2vex/pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "surfactantplugin-cvebintool2vex"
+authors = [
+    {name = "Tyler Williams", email = "tyler.williams@pnnl.gov"},
+]
+description = "Surfactant binary scanner with vex creation"
+readme = "README.md"
+requires-python = ">=3.8"
+keywords = ["surfactant"]
+license = {text = "MIT License"}
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Environment :: Console",
+    "Operating System :: MacOS",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX :: Linux",
+    "License :: OSI Approved :: MIT License",
```

Comment:
This plugin might need to be licensed under the GPLv3 (or compatible license — Apache-2.0 maybe) since cve-bin-tool is (unfortunately) released under the GPLv3.